src/util.js

```diff
@@ -1545,7 +1545,14 @@ export const importText = (itemsRanked, inputText) => {
   if (numLines === 1) {
 
     const focusOffset = window.getSelection().focusOffset
-    const strippedText = strip(text)
+    let strippedText = strip(text)
+
+    if (itemsRanked[0].key !== '' && strippedText[0] !== ' ') {
+
+      strippedText = ' ' + strippedText
+
+    }
+
     const newValue = destKey.slice(0, focusOffset) + strippedText + destKey.slice(focusOffset)
```

Comment:
Rather than modifying `strippedText`, the space should be added to `newValue`. If you think about it, the new space is more part of the new value than `strippedText`. In fact, adding a space to `strippedText` would mean it wasn't really stripped at all.

Try adding it inline in the `newValue` statement using a ternary operator, e.g. `+ (a ? b : c)  +` to avoid mutations and unnecessary variables.

Reply:
I did it that way the first time but then I changed it in that way because of better readability. It makes sense what you're saying about the variable name, though. I'll try to change it during the day!

Reply:
Your initial intuition was right!

Readability is in the eye of the beholder. I find that if I try to make things more readable to everyone, then I just end up with least common denominator (i.e. most dumb) code. It's a balance though.